pkg/util/ipvs/ipvs.go

```diff
@@ -122,15 +126,44 @@ func (rs *RealServer) Equal(other *RealServer) bool {
 		rs.Port == other.Port
 }
 
-// GetRequiredIPVSModules returns the required ipvs modules for the given linux kernel version.
-func GetRequiredIPVSModules(kernelVersion *version.Version) []string {
+// GetRequiredIPVSModules returns the kernel modules needed to use IPVS; all of the
+// modules in `required` are required, plus one of the modules in `nfconntrack`.
+func GetRequiredIPVSModules() []string {
 	// "nf_conntrack_ipv4" has been removed since v4.19
 	// see https://github.com/torvalds/linux/commit/a0ae2562c6c4b2721d9fddba63b7286c13517d9f
-	if kernelVersion.LessThan(version.MustParseGeneric("4.19")) {
-		return []string{KernelModuleIPVS, KernelModuleIPVSRR, KernelModuleIPVSWRR, KernelModuleIPVSSH, KernelModuleNfConntrackIPV4}
+	return []string{KernelModuleIPVS, KernelModuleIPVSRR, KernelModuleIPVSWRR, KernelModuleIPVSSH, KernelModuleNfConntrack, KernelModuleNfConntrackIPV4}
```

Comment:
This line is getting too big and can be even bigger if we have more modules in the future, I would suggest (or similar code style):
```
return []string{
       KernelModuleIPVS,
       KernelModuleIPVSRR,
       KernelModuleIPVSWRR,
       KernelModuleIPVSSH,
       KernelModuleNfConntrack,
       KernelModuleNfConntrackIPV4
}
```
This happens to reduce the line size and also helps when adding and removing lines in a diff output. It will show only the module removed or added (instead of entire line).